.gitignore

```diff
@@ -1,5 +1,5 @@
 /bower_components/
 /node_modules/
 /output/
-/.psci*
+/.psc*
```

Comment:
I changed this because of the generated `.psc-ide-port` which slips through the .gitignore otherwise.